CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Demos and examples 🤓 can be found on [fabricjs.com](http://fabricjs.com/demos
 ## 🏷️ Issue Tracker
 
 - **Before You Begin** 🎬
-  - 📌 Make sure you didn't fall into a known [**GOTCHA**](http://fabricjs.com/fabric-gotchas)
+  - 📌 Make sure you didn't fall into a known [**GOTCHA**](./GOTCHAS.md)
```

Comment:
i would like to leave the link to the website. The money from the ads corner ( is not even a banner ) can be useful at some point